src/evict/evict_lru.c

```diff
@@ -752,7 +749,8 @@ __evict_pass(WT_SESSION_IMPL *session)
          */
         if (eviction_progress == cache->eviction_progress) {
             if (WT_CLOCKDIFF_MS(time_now, time_prev) >= 20 &&
-              F_ISSET(cache, WT_CACHE_EVICT_CLEAN_HARD | WT_CACHE_EVICT_DIRTY_HARD)) {
+              F_ISSET(cache, WT_CACHE_EVICT_CLEAN_HARD | WT_CACHE_EVICT_DIRTY_HARD |
+                    WT_CACHE_EVICT_UPDATES_HARD)) {
```

Comment:
This set of things probably deserves a macro (or something) now that there are three.